roles/simulator/tasks/start.yml

```diff
@@ -56,12 +56,19 @@
     ps aux | grep "\-\-cfg\ /tmp/gnbsim" | wc -l
   register: async_result
   until: async_result.stdout == "0"
-  retries: 500
+  retries: 100
```

Comment:
Do not forget to run several tests about this to make sure that 100 is sufficient (not too short) for the case that you have all profiles enabled and multiples IMSIs running sequentially

Reply:
It waits for 5 seconds before retrying, making the total time around 8 minutes. I think this should be sufficient.